instrumentation-core/spring/spring-web-3.1/README.md

```diff
@@ -0,0 +1,86 @@
+# Manual Instrumentation for Spring-Web
+
+Provides OpenTelemetry instrumentation for Spring's RestTemplate.
+
+## Quickstart
+
+### Add these dependencies to your project.
+
+Replace `SPRING_VERSION` with the version of spring you're using.
+`Minimum version: 3.1`
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://mvnrepository.com/artifact/io.opentelemetry).
+`Minimum version: 0.8.0`
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <!-- opentelemetry -->
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-web-3.1</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+   <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-sdk</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+  <!-- spring web -->
+  <dependency>
+    <groupId>org.springframework</groupId>
+    <artifactId>spring-web</artifactId>
+    <version>SPRING_VERSION</version>
```

Comment:
Since this is part of the user's app maybe we don't need to document it. But if we do, isn't it compile scope?

Reply:
Setting the spring-web dependency to runtime works since the instrument-core:spring:spring-web project already has spring compile only dependencies. But you're right this can be confusing. I'll remove the compile scope.

I think it's important to document the spring-web dependency in this readme. I don't see a case where a user would add this artifact to their project and not already have spring-web in their build/pom file. However if this dependency is missing this project will throw a ClassDefNotFoundException. I'll add a note to clear this up.